lib/protoparser/common/lines_reader.go

```diff
@@ -37,7 +37,7 @@ func ReadLinesBlock(r io.Reader, dstBuf, tailBuf []byte) ([]byte, []byte, error)
 // maxLineLen limits the maximum length of a single line.
 //
 // It is expected that read timeout on r exceeds 1 second.
-func ReadLinesBlockExt(r io.Reader, dstBuf, tailBuf []byte, maxLineLen int) ([]byte, []byte, error) {
+func ReadBlockExt(r io.Reader, dstBuf, tailBuf []byte, maxLineLen int, isBinary bool) ([]byte, []byte, error) {
```

Comment:
There is no reason to use this function if content is binary. 

 Just use io.ReadAll instead.

Reply:
Actually it's better to use 
```
     lr := io.LimitReader(ctx.br, int64(MaxInsertRequestSize.N)+1)
    reqLen, err := ctx.reqBuf.ReadFrom(lr)
```

Reply:
using io.ReadAll